core/templates/dev/head/pages/exploration_player/information_card_modal_directive.html

```diff
@@ -8,9 +8,10 @@ <h2 class="oppia-info-card-title"><[explorationTitle]></h2>
     <p ng-if="objective">
       <[objective | truncateAndCapitalize: 95]>
     </p>
-    <p ng-if="!objective" translate="I18N_PLAYER_NO_OBJECTIVE">No objective specified.</p>
+    <p ng-if="!objective && !isExplorationPrivate" translate="I18N_PLAYER_NO_OBJECTIVE">No objective specified.</p>
```

Comment:
This shouldn't require a check on isExplorationPrivate. The objective (or lack of one) should be shown in any case.

Reply:
Yeah! misunderstood this from your previous comment.